src/rgw/rgw_admin.cc

```diff
@@ -2734,6 +2760,10 @@ int main(int argc, char **argv)
       break;
     case OPT_REALM_PULL:
       {
+        if (url.empty()) {
+          cerr << "A --url must be provided." << std::endl;
+          return EINVAL;
+        }
```

Comment:
+1 Thanks for getting this, I was  about to ask are there any cases a realm pull can even work without a `--url`